src/components/NcButton/NcButton.vue

```diff
@@ -591,7 +667,8 @@ export default {
 
 	// Icon and text button
 	&--icon-and-text {
-		padding: 0 16px 0 4px;
+		padding-block: 0;
+		padding-inline: 4px 16px;
```

Comment:
💙 for rtl friendly properties

Reply:
This would be even better via variables :)
```suggestion
		padding-block: 0;
		padding-inline: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 4);
```

Reply:
Wow, we have spacing variables now!

Reply:
Thank you, changed this to variables :)